public/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "short_name": "vulcan",
+  "name": "Vulcan",
+  "start_url": "/",
```

Comment:
Since we are in the Vulcan repo, this along with the images will probably not be included in the applications. I think it's ok to let it here as an example though.